Obsidian/Client.cs

```diff
@@ -315,24 +315,28 @@ public async Task StartConnectionAsync()
                 case ClientState.Configuration:
                     Debug.Assert(Player is not null);
 
-                    var configurationPacketReceived = new PacketReceivedEventArgs(Player, this.server, id, data);
+                    //await this.server.packetReceived.InvokeAsync(configurationPacketReceived);
```

Comment:
Why is this commented out?

Reply:
We don't use this atm and plugins can't see that event either due to the IPacket interface not being exposed